src/Makefile

```diff
@@ -235,26 +236,26 @@ $(IOS_BUILD_DIR)/reference/Xamarin.iOS.dll.mdb: $(IOS_BUILD_DIR)/reference/Xamar
 	@touch $@
 
 %.dll: $(IOS_BUILD_DIR)/compat/%.dll
-	$(Q) cp $< $<.mdb .
+	$(Q) cp $< $(<:.dll=.pdb)
```

Comment:
You're missing a dot at the end here, so you'll copy the dll over the pdb, instead of copying both the dll and the pdb to the current directory.